base_bypass_record_rules_by_context/models/ir_rule.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# © 2017 innoviù Srl <http://www.innoviu.com>
+# © 2017 Agile Business Group Sagl <http://www.agilebg.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, models
+
+
+class IrRule(models.Model):
+    _inherit = 'ir.rule'
+
+    @api.model
+    def domain_get(self, model_name, mode='read'):
+        if 'bypass_record_rules' in self._context:
```

Comment:
this basically turns off record rules, because anybody can set contexts. Very bad idea. I think this module is conceptually wrong and shouldn't be merged.